reconcile/change_owners/change_types.py

```diff
@@ -725,13 +725,37 @@ def allowed_changed_paths(
         ChangeTypeV1. the paths are represented as jsonpath expressions pinpointing
         the root element that can be changed
         """
+        paths = []
+        paths.extend(
+            self._allowed_changed_paths_for_file_type_and_schema(
+                file_ref.file_type, file_ref.schema, file_content, ctx
+            )
+        )
+        if file_ref.schema is not None:
```

Comment:
don't we need to check the change type context schema there as well ?

Reply:
in this case not. if a file_ref with a schema comes in, we need to also check path expressions of the change-type that are not bound to any schema (which would be the change-schema anyways).

Reply:
i simplified the code a bit